BitcoinApp/Application/Model/Enums.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  Enums.swift
+//  BitcoinApp
+//
+//  Created by Zhassulan Aimukhambetov on 12/5/19.
+//  Copyright © 2019 Zhassulan Aimukhambetov. All rights reserved.
+//
+
+import Foundation
+
+enum Currency: Int {
+    case KZT
+    case USD
+    case EUR
+    
+    var name: String {
+        switch self {
+        case .USD: return "USD"
```

Comment:
You may set `enum Currency: String` to enable default String conversion with `rawValue`